### ISSUE TITLE
Verify does not work when the method under verification has more than 1 argument

### ISSUE THREAD

**Issue by Jnforja** [NagRock/ts-mockito#4]
Verify does not work when the method under verification has more than 1 argument. This is because in the method isApplicable of the class MethodAction the variable index is not being incremented.

Bellow is the method isApplicable already fixed.
```javascript
 public isApplicable(methodName:string, matchers): boolean {
        if (this.methodName != methodName) {
            return false;
        }
        let allValid = true;
        let index: number = 0;
        for (let arg of this.args) {
            if (matchers[index] && !matchers[index].match(arg)) {
                allValid = false;
            }
            index++;
        }
        return allValid;
    }
```

**Comment by NagRock** [comment]
Thx, I will check that and fix. For next time you can create pull request :)

**Comment by NagRock** [comment]
Fixed: https://github.com/NagRock/ts-mockito/commit/1e6e369ce22566b362f7d2143b3df56b2c3cda23

And released v0.1.10

Please close issue if its working correctly now.
